RHDEVDOCS 4040 and 4041 rename CRC and CRW (#2329)
codenvy/che-docs: modules/administration-guide/pages/installing-che-on-red-hat-openshift-local.adoc

:_content-type: ASSEMBLY

> :navtitle: Installing {prod-short} on Red Hat OpenShift Local

Reviewer: more `{rh-os-local}`

> :keywords: overview, installing-che-on-red-hat-open-shift-local

Reviewer 2: ```suggestion
:keywords: overview, installing, installation, local
```

> :page-aliases: installation-guide:installing-che-on-red-hat-open-shift-local, overview:installing-che-on-red-hat-open-shift-local

Reviewer 2: * Always keep the old aliases, as they ensure redirection from the older publication location
* Always add an alias when you rename a page

```suggestion
:page-aliases: installation-guide:installing-che-on-codeready-containers, overview:installing-che-on-codeready-containers, installing-che-on-codeready-containers
```

:parent-context-of-installing-che-on-red-hat-open-shift-local: {context}

[id="installing-{prod-id-short}-on-red-hat-open-shift-local_{context}"]

> = Installing {prod-short} on Red Hat OpenShift Local

Reviewer: could be another use of `{rh-os-local}`

:context: installing-{prod-id-short}-on-red-hat-open-shift-local

> This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using Red Hat OpenShift Local. To install {prod-short}, use the`{prod-cli}` command-line tool or the OpenShift web console GUI.

Reviewer: more `{rh-os-local}`

> include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]

Reviewer: broken link; page is now named with "red-hat-openshift-local", not "codeready-containers"

Author: Thank you!

> include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]

Reviewer 2: ```suggestion
include::partial$proc_using-chectl-to-install-che-on-red-hat-openshift-local.adoc[leveloffset=+1]
```

> include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]

Reviewer: broken link; page is now named with "red-hat-openshift-local", not "codeready-containers"

> include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]

Reviewer 2: ```suggestion
include::partial$proc_using-openshift-web-console-to-install-che-on-red-hat-openshift-local.adoc[leveloffset=+1]
```

:_content-type: PROCEDURE
:navtitle: Installing {prod-short} on {rh-os-local}
:keywords: overview, installing, installation, local

> :page-aliases: installation-guide:installing-che-on-codeready-containers, overview:installing-che-on-codeready-containers

Reviewer 2: ```suggestion
:page-aliases: installation-guide:installing-che-on-codeready-containers, overview:installing-che-on-codeready-containers, installing-che-on-codeready-containers
```

> :parent-context-of-installing-che-on-red-hat-openshift-local: {context}

Reviewer 2: You can safely remove this line:

```suggestion
```

[id="installing-{prod-id-short}-on-red-hat-openshift-local_{context}"]
= Installing {prod-short} on {rh-os-local}

> :context: installing-{prod-id-short}-on-red-hat-openshift-local

Reviewer 2: You can safely remove this line:

```suggestion
```
